syntax = "proto3";

import "algorithm_state.proto";
import "matrix.proto";

package spatialcmc;

message PoisRegAlgorithmState {
  bayesmix.Vector regression_coefficients = 1;
  bayesmix.AlgorithmState partition = 2;
}

message PoisRegMCMC {
  repeated PoisRegAlgorithmState state = 1;
}
